src/test/functional/features/switch-to-sole-application.feature

```diff
@@ -7,7 +7,7 @@ Feature: Switch To Sole Application
     And I click "Send for review"
     Then the page URL should be "/application-sent-for-review"
 
-  Scenario: Switching to Sole application by Applicant 1
+  Scenario: Switching to Sole application by Applicant one
```

Comment:
we use applicant 1 and applicant 2 everywhere else, so do we need to change this?

Reply:
I just did that here because when you try to run one test with grep, it picks up all the other tests in the feature which have applicant 1. a better solution would be for me to number the scenarios, let me do that instead